README.md

```diff
@@ -3,13 +3,13 @@
 </p>
 
 <p align="center">
-Rustls is a modern TLS library written in Rust.  It uses <a href = "https://github.com/briansmith/ring"><em>ring</em></a> for cryptography and <a href = "https://github.com/rustls/webpki">webpki</a> for certificate
-verification.
+Rustls is a modern TLS library written in Rust.
 </p>
 
 # Status
-Rustls is mature and widely used. While most of the API surface is stable, we expect the next
-few releases will make further changes as needed to accomodate new features or performance improvements.
+Rustls is ready for production use. We aim to maintain reasonable API surface stability
```

Comment:
Maybe strengthen this to say something more like "Rustls is used in production at many organizations and projects."?

Reply:
I wonder if we could go further and give a few concrete examples? servo? @olix0r could we mention linkerd?

Reply:
Cloudflare? Google?

Reply:
I will add something like what @djc suggested but I think we should avoid listing particular consumers here, at least for now.

Reply:
@ctz Yes it's fine to mention that Linkerd uses rustls :)